python/dgl/nn/mxnet/conv/graphconv.py

```diff
@@ -43,8 +43,13 @@ class GraphConv(gluon.Block):
         Number of input features.
     out_feats : int
         Number of output features.
-    norm : bool, optional
-        If True, the normalizer :math:`c_{ij}` is applied. Default: ``True``.
+    norm : str, optional
+        How to apply the normalizer. If is `'right'`, divide the aggregated messages
+        by each node's in-degrees, which is equivalent to averaging the received messages.
+        If is `'none'`, no normalization is applied. Default is `'both'`,
+        where the :math:`c_{ij}` in the paper is applied.
+    weight : bool, optional
+        If True, apply a linear layer after message passing.
```

Comment:
After or before?

Reply:
I should avoid using after or before. The math is f(AXW) so it does not matter whether it performs XW first or AX first.